src/ray/protobuf/gcs.proto

```diff
@@ -138,6 +138,8 @@ message ActorTableData {
   double timestamp = 13;
   // The task specification of this actor's creation task.
   TaskSpec task_spec = 14;
+  // Resource mapping ids assigned to the worker executing the task.
```

Comment:
` // Resource mapping ids acquired by the leased worker. This field is only set when this actor already has a leased worker.`

Reply:
fixed